<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <meta name="author" content="David Leon (Dawei Leng)">
  <link rel="shortcut icon" href="../img/favicon.ico">
  <title>Research Update - Targeting 2019-nCoV Portal</title>
  <link href='https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700' rel='stylesheet' type='text/css'>

  <link rel="stylesheet" href="../css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../css/theme_extra.css" type="text/css" />
  <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
  
  <script>
    // Current page data
    var mkdocs_page_name = "Research Update";
    var mkdocs_page_input_path = "research_progress.md";
    var mkdocs_page_url = null;
  </script>
  
  <script src="../js/jquery-2.1.1.min.js" defer></script>
  <script src="../js/modernizr-2.8.3.min.js" defer></script>
  <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
  <script>hljs.initHighlightingOnLoad();</script> 
  
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
      <div class="wy-side-nav-search">
        <a href=".." class="icon icon-home"> Targeting 2019-nCoV Portal</a>
        <div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
	<ul class="current">
	  
          
            <li class="toctree-l1">
		
    <a class="" href="..">Home</a>
	    </li>
          
            <li class="toctree-l1">
		
    <span class="caption-text">Data Download</span>
    <ul class="subnav">
                <li class="">
                    
    <a class="" href="../CoV_Experiment_Data/">Data Mining</a>
                </li>
                <li class="">
                    
    <a class="" href="../nCov_Targets/">Target Validation</a>
                </li>
                <li class="">
                    
    <a class="" href="../nCov_Structures/">Structure Modeling</a>
                </li>
                <li class=" current">
                    
    <a class="current" href="./">Research Update</a>
    <ul class="subnav">
            
    <li class="toctree-l3"><a href="#2020">2020</a></li>
    

    <li class="toctree-l3"><a href="#2020-news">2020 News</a></li>
    

    <li class="toctree-l3"><a href="#2018">2018</a></li>
    

    <li class="toctree-l3"><a href="#2016">2016</a></li>
    

    <li class="toctree-l3"><a href="#2014">2014</a></li>
    

    <li class="toctree-l3"><a href="#2013">2013</a></li>
    

    <li class="toctree-l3"><a href="#2008">2008</a></li>
    

    <li class="toctree-l3"><a href="#2004">2004</a></li>
    

    </ul>
                </li>
    </ul>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../contact_us/">Community Discussion</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../useful_links/">Useful Links</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../history/">History</a>
	    </li>
          
        </ul>
      </div>
      &nbsp;
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="..">Targeting 2019-nCoV Portal</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="..">Docs</a> &raquo;</li>
    
      
        
          <li>Data Download &raquo;</li>
        
      
    
    <li>Research Update</li>
    <li class="wy-breadcrumbs-aside">
      
    </li>
  </ul>
  <hr/>
</div>
          <div role="main">
            <div class="section">
              
                <h2 id="2020">2020</h2>
<p>2020, Feb 7</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_p23clpidgdl.pdf">"Potential 2019-nCoV 3C-like protease inhibitors designed using generative deep learning approaches"</a><ul>
<li>This article from Insilico Medicine utilizes its generative chemistry pipeline to design novel molecules for 2019-nCoV through several generative chemistry approaches. Inputs for this model include 2019-nCoV 3C-like protease crystal structure, homology model, co-crystalized fragment, and a protease datasets. Several structures were generated and shown in the paper's results section. The most recent data results are on their official website: insilico.com/ncov-sprint.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 6</p>
<ul>
<li><a href="http://rs.yiigle.com/yufabiao/1179575.htm">"2019-nCoV: new challenges from coronavirus"</a><ul>
<li>According to current observations, 2019-nCoV is weaker than SARS in pathogenesis, but has stronger transmission competence; it's mechanism of cross-species spread might be related with angiotensin-converting enzyme Ⅱ (ACE2), which is consistent with the receptor SARS-CoV.</li>
<li>The emergence of 2019-nCoV reminds us once again of the importance of establishing a systematic coronavirus surveillance network. It also poses new challenges to prevention and control of the emerging epidemic and rapidly responses on scientific research.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 5</p>
<ul>
<li><a href="https://www.medrxiv.org/content/10.1101/2020.02.03.20020263v1.full.pdf">"Network-based Drug Repurposing for Human Coronavirus"</a><ul>
<li>The researchers present an integrative, antiviral drug repurposing methodology implementing a systems pharmacology-based network medicine platform, quantifying the interplay between the HCoV-host interactome and drug targets in the human protein-protein interaction network.</li>
<li>Using network proximity analyses of drug targets and known HCoV-host interactions in the human protein-protein interactome, the researchers computationally identified 135 putative repurposable drugs for the potential prevention and treatment of HCoVs, and prioritized 16 potential anti-HCoV repurposable drugs (including <strong>melatonin</strong>, <strong>mercaptopurine</strong>, and <strong>sirolimus</strong>) that were further validated by enrichment analyses of drug-gene signatures and HCoV-induced transcriptomics data in human cell lines.</li>
<li>Three potential drug combinations (including sirolimus plus dactinomycin, mercaptopurine plus melatonin, and toremifene plus emodin) captured by the Complementary Exposure pattern.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.02.02.931162v2">"Genomic variance of the 2019-nCoV coronavirus"</a><ul>
<li>A phylogenetic tree was constructed of novel coronavirus as well as other coronaviridae such as bat coronavirus and SARS. The sequence similarity of &gt;99% was confirmed between all 2019-nCoV genomes, and the closest bat coronavirus genome shared a 96.2% sequence identity. A proteomic comparison with other coronaviridae was also performed and key aminoacidic differences were identified for potential antiviral strategies.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_lftppuptosar.pdf">"Learning from the Past: Possible Urgent Prevention and Treatment Options for Severe Acute
Respiratory Infections Caused by 2019-nCoV"</a><ul>
<li>The spike protein used for host cell infection although shows mutations but remains a viable target for development of biologics. Other key drug targets, including RdRp and 3CLpro, share a strikingly high (&gt;95%) homology to SARS‐CoV. This paper suggests 4 potential drug candidates: an ACE2‐based peptide, remdesivir, 3CLpro-1, and novel sulfone protease inhibitor to possibly treat patients.</li>
<li>Preprint released on January 27, 2020. Published on February 5, 2020 </li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 4</p>
<ul>
<li><a href="https://www.nature.com/articles/s41422-020-0282-0">"Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro"</a><ul>
<li>This Letter to the Editor article first identifies several existing antiviral drugs to treat related viral infections. Through standard in vitro assays, the researchers found that remdesivir and chloroquine are highly effective in the control of 2019-nCoV infection in vitro, and recommend further human patient assessment based on these drugs' previous safety track record on human patients.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.02.02.931162v2">"Machine learning-based analysis of genomes suggests associations between Wuhan 2019-nCoV and bat Betacoronaviruses"</a><ul>
<li>Machine learned models and digital signal processing were used to classify 2019-nCoV genomes. After mapping using a two-dimensional numerical representation, the magnitude spectra was computed using a Discrete Fourier Transform. Afterwards, a distance matrix was calculated using Pearson Correlation Coefficients and were used as features for different machine learning algorithms. Through 10-fold cross validation, the classification accuracy for coronavirus was over 90%, indicating that 2019-nCoV can be classified closest to sarbecovirus within betacoronavirus.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_bpt2ncard.pdf">"Baricitinib as potential treatment for 2019-nCoV acute respiratory disease"</a><ul>
<li>Using BenevolentAI knowledge graph to search for approved drugs that could help 2019-nCoV, baricitinib is predicted to reduce the ability of the virus to infect lung cells. This study focuses on the disruption of AAK1 which might interrupt the passage of the virus into cells and also the intracellular assembly of virus particles. One of the six high-affinity AAK1-binding drugs was baricitinib, and this may have potential to reduce both the viral entry and the inflammation in patients.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.01.30.927889v1">Machine intelligence design of 2019-nCoV drugs</a><ul>
<li>A family of potential 2019-nCoV drugs generated by a machine intelligence-based generative network complex (GNC) is reported in this study. A SARS-CoV protease inhibitor dataset from ChEMBL, and a binding affinity training set comtaining PDBbind data were used for this model. The model prediccts 8000 potential drugs evaluated based on a latent space binding predictor and a 2D fingerprint predictor. Drug candidates are further evaluated by deep learning models, and logP, logS, and SA score were computed. These results indicate that anti-2019-nCoV compounds built in this study are in the top 15 and are predicted to work better than anti-HIV drugs current used like lopinavir and ritonavir.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.02.03.933226v1">"Preliminary identification of potential vaccine targets for 2019-nCoV based on SARS-CoV immunological studies"</a><ul>
<li>This study attempts to gain insight into vaccine design against 2019-nCoV. There is high genetic similarity between this virus and SARS coronavirus. Therefore, by screening SARS-CoV-derived B cell and T cell epitopes, a set of B cell and T cell epitopes derived from spike and nucleocapsid proteins mapped identically to 2019-nCoV proteins. There were no mutations observed meaning immune targeting of these epitopes may offer protection against 2019-nCoV.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 3</p>
<ul>
<li>Gilead Sciences and 2019 Novel Coronavirus (2019-nCoV)<ul>
<li>Remdesivir from Gilead Sciences <a href="https://www.bioworld.com/articles/432804-gileads-remdesivir-enters-china-phase-iii-trial-to-fight-coronavirus">enters Phase III</a> clinical trials in China on February 3, 2020.  </li>
<li>An official <a href="https://www.gilead.com/news-and-press/company-statements/gilead-sciences-statement-on-the-company-ongoing-response-to-the-2019-new-coronavirus">news release</a> from Gilead expressing their committment to respond to this outbreak by testing their novel investigational compound remdesivir in a controlled clinical trial as a treatment for 2019-nCoV.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Feb 2</p>
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.01.31.928796v1">Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes</a><ul>
<li>Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format.</li>
<li>Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify novel coronavirus (2019-nCoV) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for 2019-nCoV. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_hipwncdla.pdf">"Host and infectivity prediction of Wuhan 2019 novel coronavirus using deep learning algorithm"</a><ul>
<li>Using deep learning algorithms, this article builds a virus host prediction model in order to predict the potential virus hosts. The built model suggests that 2019-nCoV has close infectivity with other human coronaviruses, especially SARS and MERS. Additionally, by comparing predicted infectivity patterns, bat coronaviruses are  assigned  with  more  similar  infectivity  patterns  with  2019-nCoVs. It was also suggested that mink viruses show a closer infectivity pattern to 2019-nCov showing that bat and mink may be two candidate reservoirs  of  2019-nCov.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.01.31.929547v1">Predicting commercially available antiviral drugs that may act on the novel coronavirus (2019-nCoV), Wuhan, China through a drug-target interaction deep learning model</a><ul>
<li>We used our pre-trained deep learning-based drug-target interaction model called Molecule Transformer-Drug Target Interaction (MT-DTI) to identify commercially available drugs that could act on viral proteins of 2019-nCoV.</li>
<li>The result showed that atazanavir, an antiretroviral medication used to treat and prevent the human immunodeficiency virus (HIV), is the best chemical compound, showing a inhibitory potency with Kd of 94.94 nM against the 2019-nCoV 3C-like proteinase, followed by efavirenz (199.17 nM), ritonavir (204.05 nM), and dolutegravir (336.91 nM). </li>
<li>Lopinavir, ritonavir, and darunavir are all designed to target viral proteinases. However, in our prediction, they may also bind to the replication complex components of 2019-nCoV with an inhibitory potency with Kd &lt; 1000 nM.</li>
<li>In addition, we also found that several antiviral agents, such as Kaletra, could be used for the treatment of 2019-nCoV, although there is no real-world evidence supporting the prediction.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Jan 31</p>
<ul>
<li><a href="https://www.jstage.jst.go.jp/article/bst/advpub/0/advpub_2020.01020/_article">Drug treatment options for the 2019-new coronavirus (2019-nCoV)</a><ul>
<li>Lopinavir /Ritonavir, Nucleoside analogues, Neuraminidase inhibitors, Remdesivir, peptide (EK1), abidol, RNA synthesis inhibitors (such as TDF, 3TC), anti-inflammatory drugs (such as hormones and other molecules), Chinese traditional medicine, such ShuFengJieDu Capsules and Lianhuaqingwen Capsule, could be the drug treatment options for 2019-nCoV. However, the efficacy and safety of these drugs for 2019- nCoV still need to be further confirmed by clinical experiments</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_tsostncowc.pdf">"Therapeutic strategies in an outbreak scenario to treat the novel coronavirus originating in Wuhan, China"</a><ul>
<li>Potential treatment options for 2019-nCoV are reviewed in this article including neutralizaing antibodies, oligonucleotides, antiviral medications, and passive antibody transfer. Additionally, this article advocates for the development of a biologic that blocks 2019-nCoV entry using the ACE2 viral receptor fused to an immunoglobulin Fc domain. The sequence of the ACE2-Fc protein is also provided in this article's Figure 2.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_evncov.pdf">"Evolution and variation of 2019-novel coronavirus"</a><ul>
<li>This article discusses the variation of the 2019-nCoV. The genomes were downloaded and analyzed with bioinformatics tools. It was found that through this analysis, there are at least two different viral strains of 2019-nCoV are involved in this outbreak, and the most recent common ancestor of 2019-nCoVs appeared about 0.253-0.594 year before the epidemic.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Jan 29</p>
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.01.28.922922v2">Therapeutic Drugs Targeting 2019-nCoV Main Protease by High-Throughput Screening</a><ul>
<li>The main protease of SARS-CoV is essential for the life cycle of the virus, which showed 96.1% of similarity with the main proteaseof 2019-nCoV, is considered to be an attractive target for drug development.</li>
<li>In this study, we have identified 4 small molecular drugs with high binding capacity with SARS-CoV main protease by high-throughput screening based on the 8,000 clinical drug libraries, all these drugs have been widely used in clinical applications with guaranteed safety, which may serve as promising candidates to treat the infection of 2019-nCoV.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.01.29.924100v1">Potential inhibitors for 2019-nCoV coronavirus M protease from clinically approved medicines</a><ul>
<li>Basing on the structural information, we have predicted a list of commercial medicines which may function as inhibitors for 2019-nCoV by targeting its main protease Mpro. These drugs may also be effective for other coronaviruses with similar Mpro binding sites and pocket structures.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_cpppmnci.pdf">"Cell Pyroptosis, a Potential Pathogenic Mechanism of 2019-nCoV Infection"</a><ul>
<li>The initial infection characteristics of 2019-nCoV has been reported, including the symptoms and blood test results. Cell pyroptosis is a novel inflammatory form of programmed cell death, which has been largely studied recently. This article discusses the relationship between 2019-nCoV infection and cell pyroptosis and hypothesizes that 2019-nCoV is likely to cause cell pyroptosis, especially in lymphocytes, through the activation of NLRP3 inflammasome. </li>
</ul>
</li>
</ul>
<hr />
<p>2020, Jan 28</p>
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/2020.01.28.923011v1">Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</a><ul>
<li>We report for the first time that a SARS-CoV-specific human monoclonal antibody, CR3022, could bind potently with 2019-nCoV RBD (KD of 6.3 nM). The epitope of CR3022 does not overlap with the ACE2 binding site within 2019-nCoV RBD. Therefore, CR3022 has the potential to be developed as candidate therapeutics, alone or in combination with other neutralizing antibodies, for the prevention and treatment of 2019-nCoV infections</li>
<li>Some of the most potent SARS-CoV-specific neutralizing antibodies (e.g., m396, CR3014) that target the ACE2 binding site of SARS-CoV failed to bind 2019-nCoV spike protein, indicating that the difference in the RBD of SARS-CoV and 2019-nCoV has a critical impact for the cross-reactivity of neutralizing antibodies, and that it is still necessary to develop novel monoclonal antibodies that could bind specifically to 2019-nCoV RBD</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_nwppincmp.pdf">"Nelfinavir was predicted to be a potential inhibitor of 2019-nCov main protease by an integrative approach combining homology modelling, molecular docking and binding free energy calculation"</a><ul>
<li>This article shares progress regarding building homology models and docking small molecule drugs onto this structure. Based on the results, there were several drugs identified based on free energy calculations. Both MM/GBSA and SIE methods voted for <strong>nelfinavir</strong>, with the binding free energy of -24.69±0.52 kcal/mol and -9.42±0.04 kcal/mol, respectively.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Jan 27</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_fswncoseep.pdf">"From SARS-CoV to Wuhan 2019-nCoV Outbreak: Similarity of Early Epidemic and Prediction of Future Trends"</a><ul>
<li>The ongoing outbreak in China caused by the 2019-nCoV is very similar in many aspects  to SARS in 2003. This article, using epidemiological surveys and analyses from the early stage of the SARS outbreak, assesses and compares characteristics of these two outbreaks. Using the current data, the cumulative counts of 2019-nCoV cases was estimated about 2-3 times the total number of SARS, and the peak incidence is predicted to be in early or middle February. It is suggested that regional migration should be limited or prohibited to prevent the emergence and movement of a super-spreader. </li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_screpace2.pdf">"Single-cell RNA expression profiling of ACE2, the putative receptor of Wuhan 2019-nCov"</a><ul>
<li>This article goes into detail about the ACE2 receptor and its relation with the Wuhan 2019-nCoV. The results of thsi study indicates that ACE2 receptor expression is concentrated in a small population of type II alveolar cells, and these cells are also highly expressed with many other genes that positively regulate viral reproduction and transmission. This information may be of further use for future anti-ACE2 therapeutic research and development.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Jan 24</p>
<ul>
<li>
<p><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_afcpancippt.pdf">"A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster"</a></p>
<ul>
<li>The results of this study report the findings of 5 patients in a family cluster who were infected with the novel coronavirus. This study includes epidemiological, clinical, laboratory, radiological, and microbiological data regarding the patients. It was found that symptoms presented 3-6 days after exposure. Additionally, it was found through next-generation sequencing that the novel coronavirus is highly related to SARS coronaviruses.</li>
</ul>
</li>
<li>
<p><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_cfpincv.pdf">"Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China"</a></p>
<ul>
<li>This article shares results about clinical findings regarding the 2019-nCoV. Common symptoms reported were fever, cough, myalgia, or fatigue. In was interpreted that this new infection causes severe respiratory illness similar to the SARS outbreak previously. </li>
</ul>
</li>
</ul>
<hr />
<p>2020, Jan 22</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_facerulbbc.pdf">"Functional assessment of cell entry and receptor usage for lineage B β-coronaviruses, including 2019-nCoV"</a><ul>
<li>This article shares a recent development where researchers developed an approach to rapidly screen lineage B betacoronaviruses, and their ability to infect cell types from different species. They found that bypassing the host protease processing barrier allows several lineage B viruses to enter human cells. These different lineage B viruses can also recombine to gain entry into human cells with human ACE2 being the receptor for 2019-nCoV.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_gpsmadoi.pdf">"Genomic and protein structure modelling analysis depicts the origin and infectivity of 2019-nCoV, a new coronavirus which caused a pneumonia outbreak in Wuhan, China"</a><ul>
<li>The 2019-nCoV is revealed to be a new type of bat coronavirus and is genetically distant from the human SARS coronavirus. Its spike protein only binds weakly to the ACE2 receptor on human cells.</li>
</ul>
</li>
</ul>
<hr />
<p>2020, Jan 21</p>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_encwomsp.pdf">"Evolution of the novel coronavirus from the ongoing Wuhan outbreak and modeling of its spike protein for risk of human transmission"</a><ul>
<li>Analysis showed that the Wuhan CoV shared a common ancestor with SARS coronavirus that resembles the bat coronavirus HKU9-1. This work points to the RBD domain of the Wuhan CoV S-protein which supports strong interaction with human ACE2 molecules.</li>
</ul>
</li>
</ul>
<hr />
<h2 id="2020-news">2020 News</h2>
<ul>
<li>Summary of 2019-nCoV research progress <ul>
<li>This <a href="https://mp.weixin.qq.com/s/bYd1y9f_Rif7N9VhmlZTTQ">article</a> summarizes the recent advances made (as of January 29, 2020) regarding the 2019-nCoV outbreak. Many recent scientific publications are shared in this article.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li>Wuhan Institute of Virology, CAS identifies three potential drugs<ul>
<li>This <a href="https://www.guancha.cn/politics/2020_01_29_533574.shtml">article</a> reports that the CAS Wuhan Institute of Virology has recently identified three potential drugs that can be used against 2019-nCoV. The three drugs identified are Remdesivir GS-5734, Chloroquine Sigma-C6628, and Ritonavir. All three identified are old drugs that are currently awaiting approval for this new usage.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li>Chinese Academy of Sciences Shanghai Institute of Materia Medica and ShanghaiTech University release a list of compounds which can have an effect on 2019-nCoV<ul>
<li>This official <a href="https://mp.weixin.qq.com/s/SyTjhfInWpLGhBpf2cCT8Q">article</a> from CAS released a list of pharmaceutical compounds that may have a therapeutic effect on this new virus. This compounds come from old repurposed drugs or Traditional Chinese Medicine.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li>National Microbiology Data Center 2019-nCov Data    <ul>
<li>This is a <a href="http://nmdc.cn/#/nCoV">link</a> to the National Microbiology Data Center official website's information about the 2019-nCov virus. The official English name of this virus is C-Tan-nCov Wuhan strain 01. The link includes relevant information and images.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li>Chinese Academy of Sciences Shanghai Institute of Materia Medica and ShanghaiTech University release the 2019-nCoV 3CL protein crystal structure<ul>
<li>This team at CAS Shanghai Institute of Materia Medica and ShanghaiTech University <a href="http://wenhui.whb.cn/zhuzhanapp/kjwz/20200126/317749.html?state=123&amp;timestamp=1580193786547">released</a> the protein crystal structure for the 2019-nCoV 3CL protein. The Protein Data Bank ID is 6LU7. </li>
</ul>
</li>
</ul>
<hr />
<ul>
<li>"Shenyang Pharmaceutical University, Huazhong University of Science and Technology, and Academy of Military Medical Sciences identified pharmaceuticals that can possibly treat 2019-nCoV"    <ul>
<li>This <a href="https://mp.weixin.qq.com/s/TlI0d4XsfrrFtYHhWI7h6g">article</a> reveals several pharmaceuticals that are candidates to be used to treat the recent 2019-nCoV outbreak. </li>
</ul>
</li>
</ul>
<hr />
<h2 id="2018">2018</h2>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_fsadshku2.pdf">"Fatal swine acute diarrhoea syndrome caused by an HKU2-related coronavirus of bat origin"</a><ul>
<li>This article provides evidence that a bat coronavirus and swine acute diarrhoea syndrome coronavirus is the aetiological agent that was responsible for a large outbreak resulting in the fatal disease of pigs in China. The SADS and SARS outbreaks had similarities in geographical, temporal, ecological, and aetiological settings.</li>
</ul>
</li>
</ul>
<hr />
<h2 id="2016">2016</h2>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_cddato.pdf">"Coronaviruses — drug discovery and therapeutic options"</a><ul>
<li>Review of epidemiology, virology, clinical features, and current treatment strategies of SARS and MERS in 2016. This review article also includes a discussion of the drug discovery and development of new therapeutic options for CoV infections.</li>
</ul>
</li>
</ul>
<hr />
<h2 id="2014">2014</h2>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_cbasacdcsbsi.pdf">"Cell-based antiviral screening against coronaviruses: Developing virus-specific and broad-spectrum inhibitors"</a><ul>
<li>A 2014 review explaining cell-based assays using infectious virus (BSL-3) and surrogate assays (BSL-2) that can be implemented to accelerate antiviral development against SARS, MERS, and future emergent coronaviruses such as the recent 2019-nCoV.</li>
</ul>
</li>
</ul>
<hr />
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_sarsmers.pdf">"Severe acute respiratory syndrome vs. the Middle East respiratory syndrome"</a><ul>
<li>This review compares clinical features, laboratory aspects, and treatment options of SARS-like coronaviruses which typically come from bats. This article compares the similarities and differences of SARS to MERS.</li>
</ul>
</li>
</ul>
<hr />
<h2 id="2013">2013</h2>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_nisarscetatdm.pdf">"Novel Inhibitors of Severe Acute Respiratory Syndrome Coronavirus Entry That Act by Three Distinct Mechanisms."</a><ul>
<li>There are no approved treatments as of this article for SARS coronavirus. This article reports the discovery and characterization of several small-molecule inhibitors of SARS-CoV replication through several mechanisms. It showed that their mechanisms of action revealed that the compounds act by three distinct mechanisms: (i) SSAA09E2 {<em>N</em>-[[4-(4-methylpiperazin-1-yl)phenyl]methyl]-1,2-oxazole-5-carboxamide} acts through a novel mechanism of action, by blocking early interactions of SARS-S with the receptor for SARS-CoV, angiotensin converting enzyme 2 (ACE2); (ii) SSAA09E1 {[(<em>Z</em>)-1-thiophen-2-ylethylideneamino]thiourea} acts later, by blocking cathepsin L, a host protease required for processing of SARS-S during viral entry; and (iii) SSAA09E3 [<em>N</em>-(9,10-dioxo-9,10-dihydroanthracen-2-yl)benzamide] also acts later and does not affect interactions of SARS-S with ACE2 or the enzymatic functions of cathepsin L but prevents fusion of the viral membrane with the host cellular membrane.</li>
</ul>
</li>
</ul>
<hr />
<h2 id="2008">2008</h2>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_sarscap.pdf">"SARS coronavirus Accessory Proteins"</a><ul>
<li>Evidence points to the fact that accessory proteins may play an important role in modulating the host response to virus infection and contribute to pathogenesis. This review compiled the existing knowledge SARS-CoV accessory proteins and attempts to explain the possible roles of these proteins in the propagation and virulence of SARS-CoV in its host. </li>
</ul>
</li>
</ul>
<hr />
<h2 id="2004">2004</h2>
<ul>
<li><a href="http://ghddiai.oss-cn-zhangjiakou.aliyuncs.com/file/file_sbdnace2i.pdf">"Structure-Based Discovery of a Novel Angiotensin- Converting Enzyme 2 Inhibitor"</a><ul>
<li>The Angiotensin-converting enzyme 2 (ACE2) is an important therapeutic target for cardiovascular diseases and SARS outbreaks. This study identifies the molecules with the highest predicted binding scores and assays them for ACE2 enzymatic inhibitory activity and for SARS coronavirus spike protein-mediated cell fusion inhibition.</li>
</ul>
</li>
</ul>
              
            </div>
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="../contact_us/" class="btn btn-neutral float-right" title="Community Discussion">Next <span class="icon icon-circle-arrow-right"></span></a>
      
      
        <a href="../nCov_Structures/" class="btn btn-neutral" title="Structure Modeling"><span class="icon icon-circle-arrow-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="http://www.mkdocs.org">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
      
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" style="cursor: pointer">
    <span class="rst-current-version" data-toggle="rst-current-version">
      
      
        <span><a href="../nCov_Structures/" style="color: #fcfcfc;">&laquo; Previous</a></span>
      
      
        <span style="margin-left: 15px"><a href="../contact_us/" style="color: #fcfcfc">Next &raquo;</a></span>
      
    </span>
</div>
    <script>var base_url = '..';</script>
    <script src="../js/theme.js" defer></script>
      <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" defer></script>
      <script src="../search/main.js" defer></script>

</body>
</html>
